include/vara/Feature/FeatureModelTransaction.h

```diff
@@ -682,7 +682,9 @@ class AddChild : public FeatureModelModification {
                          Parent);
     assert(C && P);
 
-    removeEdge(*C->getParent(), *C);
+    if (C->getParent()) {
+      removeEdge(*C->getParent(), *C);
+    }
```

Comment:
Is this a special case where we remove something from root? (so: `C == Root`)?

Reply:
Yes, exactly that is the intention. Additionally we are now more robust regarding removal of parents beforehand, as they are reassigned in this step anyway.